hw3/01-characters.js

```diff
@@ -1,4 +1,32 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
+const charactersElement = document.getElementById("characters");
 
-fetch(url);
+fetch(url)
+  .then((response) => response.json())
+  .then((characters) => {
+    characters.forEach((character) => {
+      const characterElement = document.createElement("div");
+      characterElement.classList.add("character");
+
+      const characterImage = document.createElement("img");
+      characterImage.src = character.imageUrl;
+      characterImage.alt = `${character.fullName} image`;
+      characterImage.classList.add("character-image");
+
+      const characterName = document.createElement("h2");
+      characterName.textContent = character.fullName;
+      characterName.classList.add("character-name");
+
+      const characterTitle = document.createElement("h3");
+      characterTitle.textContent = character.title;
+      characterTitle.classList.add("character-title");
+
+      characterElement.appendChild(characterImage);
+      characterElement.appendChild(characterName);
+      characterElement.appendChild(characterTitle);
```

Comment:
Using the regular append() method all three child elements (characterIage, characterName, and characterTitle) could be appended to characterElement at the same time. This is not necessary, but it would save a few lines while still being readable.

The append() method works like appendChild(), but lets you pass in multiple items.